src/App.tsx

```diff
@@ -47,25 +49,41 @@ function App() {
   };
 
   const addCard = () => {
-    const newCard = <NewCard editMode={editMode} setEditMode={setEditMode} />;
+    setDisplayNewCard(true);
+  };
 
-    console.log("added");
+  const saveCard = ( card: any) => {
```

Comment:
**nitpick:** We should not leave any `any` in code. This object structure should be typed